File: 1-easy/problems/1560-most-visited-sector-in-a-circular-track/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def mostVisited(self, n, rounds):
        """
        :type n: int
        :type rounds: List[int]
        :rtype: List[int]
        """
        return range(rounds[0], rounds[-1]+1) or \
               range(1, rounds[-1]+1) + range(rounds[0], n+1)

File: 1-easy/problems/1560-most-visited-sector-in-a-circular-track/solution.ts
function mostVisited(n: any, rounds: any): boolean | number | string | any {
  """
  :type n: int
  :type rounds: List[int]
  :rtype: List[int]
  """
  return Array.from({length: rounds[0], rounds[-1]+1}, (_, i) => i) || \;
         Array.from({length: 1, rounds[-1]+1}, (_, i) => i) + Array.from({length: rounds[0], n+1}, (_, i) => i)
}

export { mostVisited };
